### ISSUE TITLE
Datagrid Row Context Menu Support

### ISSUE THREAD

**Issue by MelonCoffeeStudios** [blazority/support#38]
**Feature Type**
[ ] Clarity Port
[X ] Enhancement

Datagrid:
It would be so handy if we could use Blazority to attach a context menu that is generated based on the current row clicked on.
I investigated using the RowClicked event handler but this doesn't seem to register on right clicks.

Something like the below code would be a good way to mock up the Context Menu item:
```razor
<Datagrid TRowData="BlueprintDocument" Items="documents" RowClicked="(u) => DocumentClicked.InvokeAsync(u)" RowClass="DynamicRowClass" >

	<DatagridContextMenu Context="selectedItem">
		<Item OnClick="@OnClick(selectedItem)">Item 1</Item>
		<Item OnClick="@OnClick(selectedItem)">Item 2</Item>
		<Item OnClick="@OnClick(selectedItem)" Enabled="false">Item 3 (disabled)</Item>
		<Seperator />
		<Item>Submenu
			<SubMenu>
				<Item OnClick="@OnClick(selectedItem)">Submenu Item 1</Item>
				<Item OnClick="@OnClick(selectedItem)">Submenu Item 2</Item>
			</SubMenu>
		</Item>
	</DatagridContextMenu>
        ...
```

**Comment by karan-kang** [comment]
Sure @MelonCoffeeStudios, I will queue this for upcoming release.

**Comment by karan-kang** [comment]
Hi @MelonCoffeeStudios, this is now addressed in v0.9.6.
See it in action at one of these site:
- https://61a820110bc6553ba73feb8f--blazority.netlify.app/datagrid/row-actions
- https://blazority.com/datagrid/row-actions  (note: make sure to wait for upgrade)

![image](https://user-images.githubusercontent.com/1234394/144340666-a712a15a-deb6-498c-8fdf-529f0c32dbe4.png)

**Comment by MelonCoffeeStudios** [comment]
PERFECT, thanks so much!!!
